### DISCUSSION TITLE
How can I generate Ean18 code to SVG with separator?

### DISCUSSION THREAD

**Discussion by kyuranger** [micjahn/ZXing.Net#505] [Q&A]
I want to generate Ean18 to SVG like this:
![image](https://github.com/micjahn/ZXing.Net/assets/30720036/6f4d4eb5-cf77-4c28-ac11-59aed6643485)

As you see, the code has a lead, separator and trailer.

This question is similar like https://github.com/micjahn/ZXing.Net/discussions/452

However, the solution above using some package only works in Windows and currently I need to run in Linux/MAC/Android/IOS for cross platform.

In addition, that way seems cannot generate to SVG format also.

How can I solve this?

**Comment by micjahn** [comment]
You have to write your own renderer class (similar to AlternateBitmapRenderer).
